<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="381611" started-at="2019-06-13T10:48:43Z" finished-at="2019-06-13T10:55:05Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="381611" started-at="2019-06-13T10:48:43Z" finished-at="2019-06-13T10:55:05Z">
      <class name="TestScenarios.TestHtmlunit">
        <test-method status="PASS" signature="setUp()[pri:0, instance:TestScenarios.TestHtmlunit@76a3e297]" name="setUp" is-config="true" duration-ms="12" started-at="2019-06-13T10:48:43Z" finished-at="2019-06-13T10:48:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="verifyImage()[pri:0, instance:TestScenarios.TestHtmlunit@76a3e297]" name="verifyImage" duration-ms="381502" started-at="2019-06-13T10:48:43Z" finished-at="2019-06-13T10:55:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyImage -->
        <test-method status="FAIL" signature="tearDown()[pri:0, instance:TestScenarios.TestHtmlunit@76a3e297]" name="tearDown" is-config="true" duration-ms="1" started-at="2019-06-13T10:55:05Z" finished-at="2019-06-13T10:55:05Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at TestScenarios.TestHtmlunit.tearDown(TestHtmlunit.java:154)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.afterRun(TestRunner.java:725)
at org.testng.TestRunner.run(TestRunner.java:509)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestScenarios.TestHtmlunit -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
